homework/src/App.css

```diff
@@ -0,0 +1,6 @@
+#root {
```

Comment:
You have 2 css files and in both of them styles to root are applied, maybe we could combine them in one place?